action.yml

```diff
@@ -1,10 +1,16 @@
 name: 'Setup Dotnet environment'
-description: 'Setup a Dotnet environment and add it to the PATH, additionally providing proxy support'
+description: 'Setup a dotnet environment and add it to the PATH, set up proxy support, and set up authentication to private package sources like GPR.'
 author: 'GitHub'
 inputs: 
   dotnet-version:
     description: 'SDK version to use. E.g. 2.2.104'
-# Deprecated option, do not use. Will not be supported after October 1, 2019
+  source-url:
+    description: 'Optional package source for which to set up authentication. Will consult any existing NuGet.config in the root of the repo and provide a temporary NuGet.config using the NUGET_AUTH_TOKEN environment variable as a ClearTextPassword'
+  owner:
+    description: 'Optional OWNER for using packages from GitHub Package Registry organizations/users other than the current repository''s owner. Only used if a GPR URL is also provided in source-url'
+  config-file:
+    description: 'Optional NuGet.config location, if your NuGet.config isn''t located in the root of the repo.'
```

Comment:
```suggestion
    description: 'Optional NuGet.config location, if your NuGet.config isn't located in the root of the repo.'
```

Reply:
This is intentional - it's how YAML does escaping.